site-scrapers/ReverePopup/config.js

```diff
@@ -0,0 +1,13 @@
+const site = {
+    name: "Revere Board of Health Popup Clinic",
+    street: "140 American Legion Hwy",
+    city: "Revere",
+    zip: "02151",
+    signUpLink: "https://www.maimmunizations.org//reg/4012583569",
+    restrictions: "For those who live and work in Revere",
+    extraData: "Moderna",
+};
```

Comment:
This doesn't seem to be used and is conflict with the color.com registration for this clinic.

Reply:
The Color registration clinic is the other one. This is the link to the Revere Board of Health run clinic